files/qsub.pl

```diff
@@ -583,28 +583,28 @@ sub parse_script
         }
     };
 
-    # handle queues
-    if ($orig_argsh{'q'} || $set{'q'}) {
-        warn "Please do not set a queue (\'-q\' option or \'#PBS -q\' directive).\n" .
-            "Right now it sets only the default walltime based on the queue,\n" .
-            "So please request explicitly the desired walltime.\n";
-
-    }
+    # handle queues and special queues (as a partition)
     my $userqueue = $destination || $set{'q'};
     if ($userqueue) {
-        my %modwalltime = (
-            'short'  => '1:00:00',
-            'bshort' => '1:00:00', 
-            'long'   => '72:00:00',
-            'debug'  => '15:00',
-        );
-        if ($modwalltime{$userqueue}) {
-            splice(@newtxt, 1, 0, "## walltime from deprecated queue $userqueue", "#PBS -l walltime=$modwalltime{$userqueue}");
-        } else {
-            fatal("You have used a non-existing queue name!\n");
+        if ($userqueue =~ "_special") {
+            push(@cmd, '--partition', $userqueue);
```

Comment:
no special default walltime?

Reply:
@stdweird should I give 6 days walltime as default?

Reply:
yeah, but handle the walltime like the orther queues
* do not warn for special queue
* and partiton for special queue only

Reply:
I would not use 6 days as default, that's nuts. Make it 1h by default, just like for the other queues.

Or at the very least use the default like it was before (I don't think that was 6 days, that's the *max* walltime, not the default).

Reply:
torque default was 2 days

Reply:
done